yojson.opam

```diff
@@ -1,22 +1,30 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "martin@mjambon.com"
 authors: ["Martin Jambon"]
-homepage: "https://mjambon.github.io/mjambon2016/yojson-doc/Yojson.html"
-bug-reports: "https://github.com/mjambon/yojson/issues"
-dev-repo: "https://github.com/mjambon/yojson.git"
-available: [ ocaml-version >= "4.02.3"]
+homepage: "https://github.com/ocaml-community/yojson"
+bug-reports: "https://github.com/ocaml-community/yojson/issues"
+dev-repo: "git+https://github.com/ocaml-community/yojson.git"
+doc: "https://ocaml-community.github.io/yojson/"
 build: [
   ["jbuilder" "subst" "-n" name] {pinned}
   ["jbuilder" "build" "-p" name "-j" jobs]
 ]
-
-build-test: [
-  ["jbuilder" "runtest" "-p" name]
-]
-
+run-test: ["jbuilder" "runtest" "-p" name]
 depends: [
+  "ocaml" {>= "4.02.3"}
   "jbuilder" {build}
   "cppo" {build}
   "easy-format"
   "biniou" {>= "1.2.0"}
 ]
+synopsis:
+  "Yojson is an optimized parsing and printing library for the JSON format"
+description: """
```

Comment:
Should a copy of the synopsis be at the start of the description?

Reply:
I don't know. Since there was no `descr` file I just copied the description and synopsis from the already opam2 converted `opam-repository`, trusting it'd be an accurate representation of the library.

https://github.com/ocaml/opam-repository/blob/9cd5c37b861f9ec741883f3889a89653557105b9/packages/yojson/yojson.1.4.1/opam#L18-L28

Reply:
I believe that you want the description to be standalone, so yes, the synopsis text needs to be at the start of the description as well.

Reply:
Added in ab4115f.